clients/cobol-lsp-vscode-extension/src/commands/SmartTabCommand.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+import * as vscode from "vscode";
+import { ProfileService } from "../services/ProfileService";
+
+const COBOL_SECTIONS_DEFINITION: number[] = [7, 11, 72, 80];
```

Comment:
Here should be also 6 to allow jumping to the indicator area

Reply:
Fixed